P5/antbuilder.xml

```diff
@@ -55,13 +55,40 @@
     <delete dir="Schema"/>
     <mkdir dir="Schema"/>
     
-    <xslt processor="trax" force="yes" style="${XSL}/odds/odd2json.xsl" in="p5subset.xml" out="p5subset.js">
+    <xslt processor="trax" force="yes" style="${XSL}/odds/odd2json.xsl" in="p5subset.xml" out="p5subset.json">
       <factory name="net.sf.saxon.TransformerFactoryImpl"/>
-      <param name="callback" expression="teijs"/>
```

Comment:
Just to make sure: that `callback` param is not used anywhere (thus dropped in your PR)?

Reply:
Yes, this is a remnant of the first version of odd2json, which wrapped the data in a callback, JSONP style. This parameter has been dropped for a while.

Reply:
Thanks for the clarification!